README.md

```diff
@@ -46,11 +55,37 @@ the IPython kernel when you make changes.
  - [pymc3](https://github.com/pymc-devs/pymc3) (optional)
  - [zipline](https://github.com/quantopian/zipline) (optional; requires master, *not* 0.7.0)
 
+Some of Pyfolio's functionality, such as the [Bayesian tearsheet](https://github.com/quantopian/pyfolio/blob/master/pyfolio/examples/bayesian.ipynb), requires PyMC3 and Theano. To get set up, you can run:
+
+```bash
+pip install theano
+```
+
+```bash
+pip install --process-dependency-links git+https://github.com/pymc-devs/pymc3
```

Comment:
Since that PR was merged about a minute ago, we're really keeping up with the state of the art :) updating